src/composables/currencies.ts

```diff
@@ -46,41 +61,55 @@ const initPollingWatcher = createPollingBasedOnMountedComponents(POLLING_INTERVA
 
 export function useCurrencies({
   withoutPolling = false,
+  selectedProtocol = PROTOCOL_AETERNITY, // TODO - remove default value & make the protocol required
 }: UseCurrenciesOptions = {}) {
-  const minTipAmount = computed(() => 0.01 / (currencyRates.value.usd || 1));
+  const minTipAmount = computed(() => 0.01 / (currencyRates.value?.[selectedProtocol].usd || 1));
   const currentCurrencyRate = computed(
-    (): number => currencyRates.value[currentCurrencyCode.value] || 0,
+    (): number => currencyRates.value?.[selectedProtocol]?.[currentCurrencyCode.value] || 0,
   );
   const currentCurrencyInfo = computed(
     (): ICurrency => CURRENCIES.find(({ code }) => code === currentCurrencyCode.value)!,
   );
 
-  async function loadAeternityData() {
+  async function loadCoinsData() {
     try {
-      const aeMarketData = await CoinGecko.fetchCoinMarketData(
-        AE_COINGECKO_COIN_ID,
+      const [aeMarketData, btcMarketData, dogeMarketData] = await CoinGecko.fetchCoinMarketData(
+        [AE_COINGECKO_COIN_ID, BTC_COINGECKO_COIN_ID, DOGE_COINGECKO_COIN_ID],
         currentCurrencyCode.value,
-      );
+      ) || [];
 
       aeternityData.value = {
-        ...aeMarketData || {} as any,
+        ...(aeMarketData || {}) as any,
         ...AE_TOKEN_BASE_DATA,
         convertedBalance: new BigNumber(0),
       };
+
+      bitcoinData.value = {
+        ...(btcMarketData || {}) as any,
+        ...BTC_TOKEN_BASE_DATA,
+        convertedBalance: new BigNumber(0),
+      };
+
+      dogecoinData.value = {
+        ...(dogeMarketData || {}) as any,
+        ...DOGE_TOKEN_BASE_DATA,
+        convertedBalance: new BigNumber(0),
+      };
     } catch (e) {
       handleUnknownError(e);
       aeternityData.value = undefined;
+      bitcoinData.value = undefined;
     }
   }
 
   function setCurrentCurrency(currency: CurrencyCode) {
     currentCurrencyCode.value = currency;
-    loadAeternityData();
+    loadCoinsData();
     setLocalStorageItem([LOCAL_STORAGE_CURRENCY_KEY], currency);
   }
 
   async function loadCurrencyRates() {
-    const fetchedCurrencyRates = await CoinGecko.fetchCoinCurrencyRates(AE_COINGECKO_COIN_ID);
+    const fetchedCurrencyRates = await CoinGecko.fetchCoinCurrencyRates(`${AE_COINGECKO_COIN_ID},${BTC_COINGECKO_COIN_ID},${DOGE_COINGECKO_COIN_ID}`);
```

Comment:
The list of protocols and the IDs should be dynamic. Please see this as a reference: https://github.com/aeternity/superhero-wallet/pull/2226/files#r1290074335